src/Pages/AccountsdetailsPage/AccountdetailsPage.js

```diff
@@ -0,0 +1,101 @@
+import { until } from '@lion/core';
+import { html, css, LitElement } from 'lit-element';
+import { localize, LocalizeMixin } from '@lion/localize';
+import { FeServices } from '../../FeServices.js';
+
+export class AccountdetailsPage extends LocalizeMixin(LitElement) {
+  static get localizeNamespaces() {
+    return [
+      {
+        'account-details-page': locale => import(`./translations/${locale}.js`),
+      },
+      ...super.localizeNamespaces,
+    ];
+  }
+
+  static get styles() {
+    return css`
+      .cards {
+        display: flex;
+        margin-top: 10px;
+
+        justify-content: center;
+      }
+      .footer {
+        justify-content: center;
+        margin-top: 20px;
+      }
+      .button {
+        margin-left: 100px;
+      }
+      legend {
+        font-size: 18px;
+      }
+
+      .hidden {
+        display: none;
+      }
+    `;
+  }
+
+  static get properties() {
+    return {
+      data: { type: Object },
+    };
+  }
+
+  constructor() {
+    super();
+    this.data = [];
+    this.isError = false;
+  }
+
+  firstUpdated() {
+    super.firstUpdated();
+
+    this.getAccountDetaildInformation();
+  }
+
+  async getAccountDetaildInformation() {
+    this.isError = false;
+    try {
+      const data = await FeServices.getRequest({
+        url: '/accountinfo/925548553975232',
```

Comment:
try to pass the id `925548553975232` from parent, because you will get this data once login is success